src/css/profile/mobile/common/progress.less

```diff
@@ -192,19 +192,71 @@ tau-progress {
 	top: 0;
 	height: 100%;
 	padding: 0;
-	background-color: var(--progress-bar-color);
-	mask-image: url('./images/8_Progress/progress_bar_ind.png');
-	mask-repeat: no-repeat;
+	background-color: transparent;
 	border-radius: 1.5 * @px_base;
-	animation: indeterminatebar 1200ms infinite linear;
+
+	&::before {
```

Comment:
both before and after implement dark blue sliding element
but why we need both of them?

Reply:
These two bars in the same time are show in two different places?